lib/tasks/example_data.rake

```diff
@@ -14,12 +14,16 @@ namespace :example_data do
     Faker::Config.locale = "en-GB"
     Faker::UniqueGenerator.clear
 
-    assessors.each do |assessor|
+    staff_members = admins + assessors
+
+    staff_members.each do |staff|
       FactoryBot.create(
         :staff,
         :confirmed,
-        name: assessor[:name],
-        email: assessor[:email],
+        name: staff[:name],
+        email: staff[:email],
+        award_decline_permission: staff[:award_decline_permission] || false,
+        support_console_permission: staff[:support_console_permission] || false,
```

Comment:
```suggestion
        **staff
```

This should work to automatically create a staff instance with the parameters in the `staff` hash.

Reply:
I think this is done, if you want to review it